packages/protocol/contracts/standards/IERC900.sol

```diff
@@ -2,7 +2,8 @@ pragma solidity ^0.5.17;
 
 
 // Interface for ERC900: https://eips.ethereum.org/EIPS/eip-900
-interface ERC900 {
+// Note: we should also add the optional history methods as a separate interface
```

Comment:
Why separate? [Here](https://eips.ethereum.org/EIPS/eip-900) they appear altogether.

Reply:
I was thinking separate, because our current `IERC900` interface doesn't include them... and generally optional interfaces in ERCs are now split into different ones (as the contract won't compile if you include them, and they're "optional").

For example, see [ERC-721](https://eips.ethereum.org/EIPS/eip-721).